cmd/statusd/main.go

```diff
@@ -62,6 +62,8 @@ var (
 	// Push Notification
 	enablePN     = flag.Bool("shh.notify", false, "Node is capable of sending Push Notifications")
 	firebaseAuth = flag.String("shh.firebaseauth", "", "FCM Authorization Key used for sending Push Notifications")
+
+	exitWhenSynced = flag.Bool("exit-when-synced", false, "Exit after blockchain sync")
```

Comment:
Maybe we can make it an integer and default to `-1`? If it's larger than `-1` then this option is active and its value tells how long to wait for a synchronization to finish? `0` could mean wait infinitely long until it's synced.

Reply:
Sure, shall we expect minutes for values larger than 0?

Reply:
Ah okay, it's already clear in the comment below.